docs/api/app.md

```diff
@@ -1277,7 +1277,7 @@ Returns `Object`:
   * `scope` string _Windows_ - one of `user` or `machine`. Indicates whether the registry entry is under `HKEY_CURRENT USER` or `HKEY_LOCAL_MACHINE`.
   * `enabled` boolean _Windows_ - `true` if the app registry key is startup approved and therefore shows as `enabled` in Task Manager and Windows settings.
 
-### `app.setLoginItemSettings(settings)` _macOS_ _Windows_
+### `app.setLoginItemSettings(settings)` _macOS_ _Windows_ _Linux_
```

Comment:
```suggestion
### `app.setLoginItemSettings(settings)`
```

When things are supported across all 3 major platforms, we usually just remove all platform notices.